emoji/src/main/java/com/vanniktech/emoji/EmojiManager.java

```diff
@@ -134,11 +134,16 @@ public static void install(@NonNull final EmojiProvider provider) {
     INSTANCE.emojiRepetitivePattern = Pattern.compile('(' + regex + ")+");
   }
 
-  static void destroy() {
+  public static void destroy() {
+    for (final Emoji emoji : INSTANCE.emojiMap.values()) {
```

Comment:
Let's use fori here too

Reply:
We can't; [values()](https://docs.oracle.com/javase/7/docs/api/java/util/Map.html#values()) returns a `Collection` and not a `List`.